meta-lmp-base/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -46,16 +46,6 @@ PACKAGECONFIG ?= " \
     zstd \
 "
 
-PACKAGECONFIG[p11kit] = "-Dp11kit=true,-Dp11kit=false,p11-kit"
```

Comment:
Please point the hash/commit from upstream which included this change.

Reply:
done